engine/runtime/tests/fixture/workflow/geometry/convex_hull_accumulator.yml

```
@@ -0,0 +1,47 @@
+        with:
+          format: geojson
+          inline: |
+            env.get("geojson")
+
+      - id: 61e89fd2-ea66-4fa1-b426-6f84484a9d39
+        name: ConvexHullAccumulator
+        type: action
+        action: ConvexHullAccumulator
+        with:
+          groupBy:
+          - municipality
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)